Add more examples for $search (#15083)
microsoftgraph/microsoft-graph-docs: concepts/search-query-parameter.md

## Using $search on directory object collections

> Azure AD resources and their relationships that derive from [directoryObject](/graph/api/resources/directoryobject) support the `$search` query parameter only in advanced queries. The search implementation does **not** support `contains` due to performance reasons. Instead it uses a tokenization approach which works by extracting words from your input and output string, using spaces, numbers, different casing, and symbols to separate the words, as follows:

Reviewer: Recommending that we avoid highlighting this reason
```suggestion
Azure AD resources and their relationships that derive from [directoryObject](/graph/api/resources/directoryobject) support the `$search` query parameter only in advanced queries. The search implementation does **not** support `contains` but uses a tokenization approach which works by extracting words from your input and output string, using spaces, numbers, different casing, and symbols to separate the words, as follows:
```

Author: Fixed.
